tests/devices/test_default_mixed.py

```diff
@@ -17,19 +17,53 @@
 
 import pytest
 import pennylane as qml
+
+from pennylane.ops import PauliZ, CZ, PauliX, Hadamard, CNOT, AmplitudeDamping, DepolarizingChannel
```

Comment:
(Minor suggestion): these classes seem to be referred to in different ways in different parts of the files: e.g. `qml.PauliZ` or `PauliZ`, would be nice to make it consistent (would personally suggest the `qml.PauliZ` form partly for fewer import statements)